ow_faults/src/FaultInjector.cpp

```diff
@@ -26,6 +32,20 @@ void FaultInjector::faultsConfigCb(ow_faults::FaultsConfig& faults, uint32_t lev
   m_faults = faults;
 }
 
+void FaultInjector::setSystemFaultMessage(ow_faults::SystemFaults& msg) {
+  // for now only arm execution errors
+  msg.header.stamp = ros::Time::now();
+  msg.header.frame_id = '/world';
```

Comment:
Should be "/world".  Single quotes are for characters, and this produced a compiler warning for me.

Reply:
good catch, I was wondering why that was the case. :D will fix